assets/js/modules/analytics/components/setup/SetupFormGA4.js

```diff
@@ -77,9 +78,9 @@ export default function SetupFormGA4() {
 				<GA4PropertySelect />
 			</div>
 
-			<GA4PropertyNotice
+			{ ga4PropertyID && ( ga4PropertyID !== PROPERTY_CREATE ) && <GA4PropertyNotice
```

Comment:
See https://github.com/google/site-kit-wp/issues/3614#issuecomment-875856859, only the first condition here must be kept. Only if no property selection has been made (setting up a new property _is_ a selection), the notice must not be shown.

Reply:
👍